src/state.rs

```diff
@@ -271,6 +276,31 @@ impl StateSet {
     pub fn iter(self) -> impl Iterator<Item = State> {
         self.0.iter()
     }
+
+		/// Create a StateSet, with all states active.
+		pub fn all() -> Self {
+			StateSet(State::all())
+		}
+
+		/// Checks if all states are set.
```

Comment:
I don't see any use case for this. The purpose of creating newtypes is to guide people into using the API correctly, as well as giving them the tools that make sense. A `StateSet` is not a `BitFlags`. Therefore not all methods of `BitFlags` are applicable to `StateSet`.